### PR TITLE
Redis compatibility client documentation page.

### PR DESCRIPTION
**Pull request by NoSQLKnowHow**
Please do not merge this yet. Let me merge it when I have the go-ahead, for timing purposes.

### CODE REVIEW

File: docs/develop/sdks-integrations/redis-client-compatibility.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+```
+
+  </td>
+</tr>
+</table>
+
+  </TabItem>
+</Tabs>
+
+If there is a Redis operation the Redis compatibility client do not currently support, but you need. Please message [Momento Support](mailto:support@momentohq.com).
```
Comment (cprice404) [inline docs/develop/sdks-integrations/redis-client-compatibility.md]:
```suggestion
If there is a Redis operation the Redis compatibility client do not currently support, but you need: please message [Momento Support](mailto:support@momentohq.com).
```

Comment (cprice404) [reply docs/develop/sdks-integrations/redis-client-compatibility.md]: not sure if we are also mentioning discord in support links like this these days?

**Comment by cprice404** [comment]
@NoSQLKnowHow I'm going to have a few minor changes to this based on the last bits of work I've done to the API/docs in the repo.  See this PR for a heads up:

https://github.com/momentohq/momento-node-redis-client/pull/18

File: docs/develop/sdks-integrations/redis-client-compatibility.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+import TabItem from '@theme/TabItem';
+
+# Redis compatibility client for Momento Cache and Momento Topics
+Do you have existing apps that use a Redis cache, but now you're trading up to Momento’s services? With Redis compatibility clients, there's no need to refactor your code. Instead, compatibility clients are a drop-in replacement for existing Redis clients. Change your client library to the compatibility client, change the connection information, and the core of your code stays the same.
+
+This client facilitates you using Momento Cache as your cache engine for any node.js frameworks that support a redis-backed cache, such as [Express.js](https://github.com/expressjs/express). See the [examples section](https://github.com/momentohq/momento-node-redis-client#examples) for more info.
+
+:::info
+
+Momento Cache and Momomento Topics do not support all of Redis' operations and datatypes. If a Redis operation is not supported by the Redis compatibility client, it will throw an error.
```
Comment (cprice404) [inline docs/develop/sdks-integrations/redis-client-compatibility.md]: my 2c is still that we should mention something here about our ability / willingness to add missing operations if a user needs them, but I will defer to your judgment.

File: docs/develop/sdks-integrations/redis-client-compatibility.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+});
+
+```
+
+</td>
+<td width="50%">
+
+```javascript
+// Import the Momento redis compatibility client.
+import {createClient, Momento} from 'momento-redis-client';
```
Comment (cprice404) [inline docs/develop/sdks-integrations/redis-client-compatibility.md]:
```suggestion
import {createClient, momento} from '@gomomento-poc/node-redis-client';
```

made some minor name changes in the lib and published it to npm.js.

File: docs/develop/sdks-integrations/redis-client-compatibility.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+// Initialize Momento's client.
+const redisClient = createClient(
+  new Momento.CacheClient({
```
Comment (cprice404) [inline docs/develop/sdks-integrations/redis-client-compatibility.md]:
```suggestion
  new momento.CacheClient({
```

File: docs/develop/sdks-integrations/redis-client-compatibility.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+    configuration: Momento.Configurations.Laptop.v1(),
```
Comment (cprice404) [inline docs/develop/sdks-integrations/redis-client-compatibility.md]:
```suggestion
    configuration: momento.Configurations.Laptop.v1(),
```

File: docs/develop/sdks-integrations/redis-client-compatibility.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+    credentialProvider: Momento.CredentialProvider.fromEnvironmentVariable({
```
Comment (cprice404) [inline docs/develop/sdks-integrations/redis-client-compatibility.md]:
```suggestion
    credentialProvider: momento.CredentialProvider.fromEnvironmentVariable({
```

File: docs/develop/sdks-integrations/redis-client-compatibility.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+  </td>
+</tr>
+</table>
+
+For more in-depth information, with example code, please see the [Momento Node.js Redis compatibility client on GitHub](https://github.com/momentohq/momento-node-redis-client#momento-nodejs-redis-client).
+  </TabItem>
+</Tabs>
+
+## Current Redis API Support
+This library supports the most popular Redis APIs, but does not yet support all Redis APIs. We currently support the most common APIs related to string values (GET, SET, etc.) and hash values (HGETALL, HSET, etc.). We will be adding support for additional APIs in the future. If there is a particular API that you need support for, please drop by our [Discord](https://discord.com/invite/3HkAKjUZGq) or e-mail [Momento Support](mailto:support@momentohq.com) and let us know!
```
Comment (cprice404) [inline docs/develop/sdks-integrations/redis-client-compatibility.md]:
```suggestion
This library does not yet support all Redis APIs. We currently support the most common APIs related to string values (GET, SET, etc.) and hash values (HGETALL, HSET, etc.). We will be adding support for additional APIs in the future. If there is a particular API that you need support for, please drop by our [Discord](https://discord.com/invite/3HkAKjUZGq) or e-mail [Momento Support](mailto:support@momentohq.com) and let us know!
```

**Review by cprice404 — COMMENTED** [review]
a few code changes we need to get in, and a couple of other minor questions/nits which you can take or leave.

File: docs/develop/sdks-integrations/redis-client-compatibility.md (hunk 2 above)
Comment (NoSQLKnowHow) [reply docs/develop/sdks-integrations/redis-client-compatibility.md]: I removed this info box and just moved up the section that was at the bottom about API compatibility and contacting us.

File: docs/develop/sdks-integrations/redis-client-compatibility.md (hunk 7 above)
Comment (NoSQLKnowHow) [reply docs/develop/sdks-integrations/redis-client-compatibility.md]: fixed this and uploaded a new commit.

**Review by cprice404 — APPROVED** [review]

**Pull request merged**